### PR TITLE
simplify recovery docs for beginners

### PR DESCRIPTION
**Pull request by nedpfeiffer**
As a relatively new and inexperienced Linux user, I found this recovery documentation pretty confusing. Adding a clarifying sentence to the introduction and moving around paragraphs helps simplify things. Per the Dasharo [documentation guidelines](https://docs.dasharo.com/dev-proc/documentation-guidelines/) the most important information should be at the top of the document; as such, I moved the section on the MSI Flash BIOS Button to the bottom considering it does not currently work.

If this article is more oriented towards developers it may not need any edits. Let me know what you think, thanks!

### CODE REVIEW

**Comment by miczyg1** [comment]
> If this article is more oriented towards developers it may not need any edits. Let me know what you think, thanks!

Of course the documentation is mainly for the community, not developers. Thank you for contributing!

File: docs/variants/msi_z690/recovery.md [hunk 1]
```diff
@@ -7,45 +7,7 @@ hardware configurations, the Dasharo firmware may not boot correctly (i.e.
...
-* Flash via external programmer:
-
-> The command line will be different, depending on the programmer you use
-
-```bash
-flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7D25v13/E7D25IMS.130
-```
-
-* First boot after the recovery process is significantly longer
+There are two documented recovery methods: using a CH341A programming kit or using an external programmar like an [RTE](https://3mdeb.com/open-source-hardware/#rte).
```
Comment (miczyg1) [inline docs/variants/msi_z690/recovery.md:10]: We wrap the documentation on 80 characters (except markdown syntax like links)
```suggestion
There are two documented recovery methods: using a CH341A programming kit or
using an external programmar like an [RTE](https://3mdeb.com/open-source-hardware/#rte).
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

**Comment by pietrushnic** [comment]
@nedpfeiffer please note [pre-commit checks fail](https://results.pre-commit.ci/run/github/316342881/1675843835.c8hUl7OOQhOXiHtsyZ4Q_A) for your PR. Please check [pre-commit](https://github.com/Dasharo/docs#pre-commit-hooks) section in README.

**Comment by nedpfeiffer** [comment]
This PR was kind of a mess, I'm going to close it and split it into smaller parts. Apologies.

**Comment by pietrushnic** [comment]
@nedpfeiffer sure. Thank you for your effort and we looking for improved series.

**Pull request closed without merging**